TKGame/Enemy/Enemy.cs

```diff
@@ -11,7 +11,7 @@
 using Microsoft.Xna.Framework.Media;
 using TKGame.Level_Editor_Content;
 
-namespace TKGame.Enemy
+namespace TKGame
```

Comment:
We should definitely still think about what namespace this should be in cause it doesn't feel right for it to be in the main one

Reply:
Yah, probably should create a sub directory for different enemy sub classes. Although an entities sub directory could also work if we just wanted all the entities in one namespace.